sdk/communication/Azure.Communication.CallingServer/tests.yml

```diff
@@ -18,4 +18,6 @@ extends:
     Clouds: Public,Int
     EnvVars:
       # SKIP_PHONENUMBER_LIVE_TESTS skips certain phone number tests such as purchase and release
-      SKIP_PHONENUMBER_LIVE_TESTS: TRUE
\ No newline at end of file
+      SKIP_PHONENUMBER_LIVE_TESTS: TRUE
```

Comment:
why is `SKIP_PHONENUMBER_LIVE_TESTS` here?

Reply:
I saw this flag is widely used in all the other communication SDK projects, even this flag is not used in them.